src/main/scala/devices/i2c/I2C.scala

```diff
@@ -527,10 +527,10 @@ trait HasI2CModuleContents extends MultiIOModule with HasRegMap {
 
   // statusReadReady,
   regmap(
-    I2CCtrlRegs.prescaler_lo -> Seq(RegField(8, prescaler.lo)),
-    I2CCtrlRegs.prescaler_hi -> Seq(RegField(8, prescaler.hi)),
-    I2CCtrlRegs.control      -> control.elements.map{ case(name, e) => RegField(e.getWidth, e.asInstanceOf[UInt]) }.toSeq,
-    I2CCtrlRegs.data         -> Seq(RegField(8, r = RegReadFn(receivedData),  w = RegWriteFn(transmitData))),
+    I2CCtrlRegs.prescaler_lo -> Seq(RegField(8, prescaler.lo, RegFieldDesc("prescaler_lo","I2C prescaler, low byte", reset=Some(0)))),
+    I2CCtrlRegs.prescaler_hi -> Seq(RegField(8, prescaler.hi, RegFieldDesc("prescaler_hi","I2C prescaler, high byte", reset=Some(0)))),
+    I2CCtrlRegs.control      -> control.elements.map{ case(name, e) => RegField(e.getWidth, e.asInstanceOf[UInt], RegFieldDesc("control","I2C control", reset=Some(0))) }.toSeq,
```

Comment:
should be something like: ` -> RegFieldGroup("control", "Control (or whatever this description is)", control.elements.map{ case(name, e) => RegField(e.getWidth, e.asInstanceOf[UInt], RegFieldDesc(name, s"Sets the ${name}" , reset=Some(0))) }.toSeq)`

Basically the whole register is called control, the fields are coming from the Bundle and should be named whatever they are in the bundle.